lib/canvas_yaml.rb

```diff
@@ -28,6 +28,7 @@
 if defined?(YAML::ENGINE)
```

Comment:
while you're in the area - can you confirm that this case will never happen again (I'm pretty sure it was only for Ruby 1.9, which Canvas no longer allows), and remove this dead code